### ISSUE TITLE
Mermaid diagrams of type mindmap are not rendered

### ISSUE THREAD

**Issue by bentolor** [mjbvz/vscode-markdown-mermaid#153]
Hi @mjbvz - Thanks for your extension!

Mermaid now also [supports mindmaps](https://mermaid-js.github.io/mermaid/#/mindmap), a feature originally requested in #59 .

Unfortunately vscode-markdown-mermaid doesn't seem to render `mindmap` mermaid diagrams ight now, while `gitGraph` works. 

Is it just a missing mermaid dependency update?

**Comment by bentolor** [comment]
Okay - I just see it's really brand-new. Only 12 days old. Sorry - wasn't aware on that. 
https://github.com/mermaid-js/mermaid/releases/tag/v9.2.0

**Comment by mjbvz** [comment]
Picking up mermaid 9.2.2 however it looks like mindmap support is shipped as an additional dependency

**Comment by awhitford-cip** [comment]
I thought that since Mermaid was upgraded to 9.3.0 it would be included, but I am getting this error:
```
No diagram type detected for text: mindmap
```

**Comment by isaacsanders** [comment]
I think this last step is still needed: https://mermaid.js.org/syntax/mindmap.html#integrating-with-your-library-website

**Comment by isaacsanders** [comment]
I made a PR that might get this over the finish line: https://github.com/mjbvz/vscode-markdown-mermaid/pull/174

**Comment by gs-rickchristy** [comment]
Would really appreciate this.

**Comment by gogza** [comment]
Just discovered I need this too.

**Comment by LiangWei88** [comment]
I think the 9.4.0 is ok?
I need this too

**Comment by gozes** [comment]
I need this too. Hopefully we can get `9.4.0` soon?

**Comment by gozes** [comment]
I'm not a js/TS person, however, I have opened a [PR](https://github.com/mjbvz/vscode-markdown-mermaid/pull/183) to bump the version to `9.4.0`

**Comment by mjbvz** [comment]
Actually still broken after #183 
<img width="832" alt="Screenshot 2023-03-06 at 1 27 54 PM" src="https://user-images.githubusercontent.com/12821956/223234798-d0dc935f-22ea-4f3a-81a5-7c469dcb62a8.png">

Maybe needs changes in #184  too

**Comment by mjbvz** [comment]
Should work now in 1.17.5 which was just published . Let me know if you notice any issues

**Comment by gozes** [comment]
@mjbvz I'm seeing 

```
Loading chunk 366 failed.
(error: https://file+.vscode-resource.vscode-cdn.net/Users/jsmontalvo/.vscode/extensions/bierner.markdown-mermaid-1.17.5/dist-preview/366.bundle.js)
``` 

using the example from the mermaid site

```
mindmap
  root((stmory))
    Origins
      Long history
      ::icon(fa fa-book)
      Popularisation
        British popular psychology author Tony Buzan
    Research
      On effectivness<br/>and features
      On Automatic creation
        Uses
            Creative techniques
            Strategic planning
            Argument mapping
                Test
                    Eee
    Tools
      Pen and paper
      Mermaid
````

**Comment by mjbvz** [comment]
Hm works fine for me with the official extension build:

![Screenshot 2023-03-07 at 8 23 55 AM](https://user-images.githubusercontent.com/12821956/223484517-2f5b8da0-01f1-4c75-8bbd-5a3e7b40ebd7.png)

Can you please open a new issue with your OS and VS Code version info

**Comment by isaacsanders** [comment]
This bundle 366 appears to be insecurely supplied to vscode

**Comment by mjbvz** [comment]
The file is part of the extension bundle:

![Screenshot 2023-03-07 at 11 34 39 AM](https://user-images.githubusercontent.com/12821956/223533296-e053109c-29aa-4154-be31-12f56f16e248.png)

Can you please open an issue with your OS and VS Code version info

**Comment by isaacsanders** [comment]
It works if I turn off my security settings.

**Comment by fh2levelgh** [comment]
> It works if I turn off my security settings.

What security settings did you have to change?

**Comment by DaReign** [comment]
Hi I just installed plugin and have that same issue 

No diagram type detected matching given configuration for text:
